### DISCUSSION TITLE
노드 수정과 노드 상세정보 수정을 분리하는건 어떨까요?

### DISCUSSION THREAD

**Discussion by jihye-woo** [Pin-Tree/backend#58] [Ideas]
### 현재 노드 수정페이지
![image](https://user-images.githubusercontent.com/74452069/134800778-e12c6e1d-faf3-443b-ab67-18b9a7fea795.png)

### 노드 수정과 노드 상세정보 수정을 분리하는건 어떨까요?
즉 노드를 수정할 때 **노드 이름, 색상, 위치** 뿐만 아니라 **노드의 상세 정보**도 한 번에 수정할 수 있도록 되어있는데
노드 수정과 노드 상세정보 수정을 별도로 가져가는건 어떨까요?

### 구체적인 분리 계획
조금 더 구체적으로 말씀드리면
- 노드 수정
    - 사이드 바에서 노드를 수정할 때는 이름과 컬러 혹은 위치처럼 **메인 로드맵 화면에 보여지는 노드 데이터들만을 수정**한 뒤 서버로 요청 
- 노드 상세정보 수정
    - **노드 상세정보 페이지 안**으로 들어가서 수정 버튼을 누르면 그때 **수정을 가능**하게 하고, 수정 완료시에 **단건으로 서버에 수정 요청**
    - 노드 상세정보 수정은 이미지 변경 등 **변경 사항들의 데이터가 조금 클 수도 있을 것 같아서** **노드 상세정보 수정은 단일 요청**으로 보내도 좋을 것 같습니다!

노드 상세정보 수정페이지 디자인이 아직 없어서, 프론트에서 정확히 어떤 식으로 수정이 이루어질 지는 모르겠지만
디자인이 나오기 전에 백엔드끼리 의견을 맞추고서 먼저 말씀드리면 좋을 것 같아서요!

**Comment by sanhee** [comment]
저도 노드수정과 노드상세정보 수정은 개별로 가져가는게 좋다고 생각해요.
(사실 그렇게 작업하려고 했던건..비밀입니다..ㅋㅋㅋㅋ)
토의해야한다는 걸 깜박했는데, 의견 제시해주셔서 감사해요~
